WalletWasabi.Gui/Controls/TogglePasswordBox.cs

```diff
@@ -0,0 +1,62 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+using Avalonia.Data;
+using Avalonia.Styling;
+using System;
+using ReactiveUI;
+using System.Reactive.Linq;
+
+namespace WalletWasabi.Gui.Controls
+{
+	public class TogglePasswordBox : ExtendedTextBox, IStyleable
+	{
+		Type IStyleable.StyleKey => typeof(TogglePasswordBox);
+
+		public static readonly StyledProperty<bool> IsPasswordVisibleProperty =
+			AvaloniaProperty.Register<TogglePasswordBox, bool>(nameof(IsPasswordVisible), defaultBindingMode: BindingMode.TwoWay);
+
+		public bool IsPasswordVisible
+		{
+			get => GetValue(IsPasswordVisibleProperty);
+			set => SetValue(IsPasswordVisibleProperty, value);
+		}
+
+		public TogglePasswordBox()
+		{
+			UseFloatingWatermark = true;
+			Watermark = "Password";
+			MinWidth = 400;
+
+			this.GetObservable(IsPasswordVisibleProperty)
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(x =>
+			{
+				IsPasswordVisible = x;
+			});
```

Comment:
Fix indentation of this line and the two above it to be inline with `.Subscribe(x =>`

Reply:
Fixed!